src/types/NFTInterface.ts

```diff
@@ -109,6 +109,7 @@ export type AuctionLike = {
   reservePrice?: CurrencyValue;
   // current bid is duplicated within bids
   bids: readonly AuctionBidEvent[];
+  auctionId: string | null;
```

Comment:
we typically do undefined or string now. 
can do `auctionId?: string`

Reply:
fixed